src/main/resources/org/jenkinsci/plugins/workflow/support/steps/ExecutorStep/help-label.html

```diff
@@ -1,46 +1,107 @@
 <div>
-    Computer name, label name, or any other label expression like <code>linux && 64bit</code> to restrict where this step builds.
+    Computer name, label name, or any other label expression like <code>linux &amp;&amp; 64bit</code> to restrict where this step builds.
     May be left blank, in which case any available executor is taken.
-    <!-- TODO figure out how to inline or otherwise share AbstractProject/help-label.html -->
-    <h3>Valid Operators</h3>
-    <p>
-    The following operators are supported, in the order of precedence.
-    <dl>
-        <dt>(expr)</dt>
-        <dd>parenthesis</dd>
-
-        <dt>!expr</dt>
-        <dd>negation</dd>
-
-        <dt>expr&amp;&amp;expr</dt>
-        <dd>
-            and
-        </dd>
-
-        <dt>expr||expr</dt>
-        <dd>
-            or
-        </dd>
-
-        <dt>a -> b</dt>
-        <dd>
-            "implies" operator. Equivalent to <tt>!a|b</tt>.
-            For example, <tt>windows->x64</tt> could be thought of as "if run on a Windows slave,
-            that slave must be 64bit." It still allows Jenkins to run this build on linux.
-        </dd>
-
-        <dt>a &lt;-> b</dt>
-        <dd>
-            "if and only if" operator. Equivalent to <tt>a&amp;&amp;b || !a&amp;&amp;!b</tt>.
-            For example, <tt>windows&lt;->sfbay</tt> could be thought of as "if run on a Windows slave,
-            that slave must be in the SF bay area, but if not on Windows, it must not be in the bay area."
-        </dd>
-    </dl>
-    <p>
-    All operators are left-associative (i.e., a->b->c &lt;-> (a->b)->c )
-    An expression can contain whitespace for better readability, and it'll be ignored.
-
-    <p>
-    Label names or slave names can be quoted if they contain unsafe characters. For example,
-    <tt>"jenkins-solaris (Solaris)" || "Windows 2008"</tt>
+
+  <h3>Supported operators</h3>
+  The following operators are supported, in descending order of precedence:
+  <dl>
+    <dt><tt>(expression)</tt></dt>
+    <dd>
+      parentheses — used to explicitly define the associativity of an expression
+    </dd>
+
+    <dt><tt>!expression</tt></dt>
+    <dd>
+      NOT — negation; the result of <tt>expression</tt> must <b>not</b> be true
+    </dd>
+
+    <dt><tt>a &amp;&amp; b</tt></dt>
+    <dd>
+      AND — <b>both</b> of the expressions <tt>a</tt> and <tt>b</tt> must be
+      true
+    </dd>
+
+    <dt><tt>a || b</tt></dt>
+    <dd>
+      OR — <b>either</b> of the expressions <tt>a</tt> or <tt>b</tt> may be
+      true
+    </dd>
+
+    <dt><tt>a -> b</tt></dt>
+    <dd>
+      "implies" operator — equivalent to <tt>!a || b</tt>.<br>
+      For example, <tt>windows -> x64</tt> could be thought of as "if a Windows
+      agent is used, then that agent <i>must</i> be 64-bit", while still
+      allowing this project to be executed on any agents that <i>do not</i> have
```

Comment:
It seems better to me to replace `project` with `step` throughout the file, since a Pipeline job (what I think of when I read `project`) can have multiple `node` steps, so using `project` is a little confusing.

Reply:
Actually, I think you mean `stage`

Reply:
`stage` has a different meaning, right? Maybe `block` is the best way to refer to what runs inside of the `node` `step`? CC @abayer

Reply:
And, then, that gets a bit confusing, because sometimes it's the pipeline and sometimes it's the stage.

I think the right word ends up being `context`

Reply:
`block` is probably the best option - in Declarative land, yes, behind the scenes we run a `node` step for the whole Pipeline and/or for individual `stage`s, but the `node` step isn't invoked by the user directly, so the docs for the step aren't really relevant there.